modulefiles/build_orion_intel.lua

```diff
@@ -10,7 +10,6 @@ prepend_path("MODULEPATH", "/work/noaa/da/role-da/spack-stack/modulefiles")
 
 load("stack-intel/2022.0.2")
 load("stack-intel-oneapi-mpi/2021.5.1")
-load("stack-python/3.9.7")
```

Comment:
Why was the loading of `stack-python/3.9.7` removed from the `build_orion_intel.lua` modulefile?  While running the fundamental test suite on Orion, I saw the same behavior as @gspetro-NOAA, everything passed except for `grid_RRFS_CONUS_25km_ics_NAM_lbcs_NAM_suite_GFS_v16`.  This test failed in the `run_MET_Pb2nc_obs` task with the following error message:
`/work/noaa/epic/role-epic/spack-stack/spack-stack-1.4.1/envs/unified-env/install/intel/2022.0.2/met-10.1.1-eiz3e5i/bin/pb2nc: error while loading shared libraries: libpython3.9.so.1.0: cannot open shared object file: No such file or directory`
This error message usually occurs when a module that was loaded for the build is different from the module used for running.  When I added this load back into the modulefile, the test successfully ran through to completion.

Reply:
The modification that you made at hash [387373e](https://github.com/ufs-community/ufs-srweather-app/pull/938/commits/387373e85b5fea32760138835b8bf109c158f81e) has corrected the failure of the `grid_RRFS_CONUS_25km_ics_NAM_lbcs_NAM_suite_GFS_v16` WE2E test on Orion.

Resolving conversation now.